packages/@sanity/portable-text-editor/src/editor/plugins/createWithPlaceholderBlock.ts

```diff
@@ -33,25 +34,6 @@ export function createWithPlaceholderBlock({
         ],
       }
     }
-    const {onChange} = editor
```

Comment:
Removing this code, as we have a similar code in the `useSyncValue` hook now and the code is more transparent in there.